cfme/tests/middleware/test_middleware_deployment.py

```diff
@@ -1,33 +1,48 @@
-from random import sample
-
 import pytest
+from cfme.middleware import get_random_list
 from cfme.middleware.deployment import MiddlewareDeployment
+from cfme.middleware.server import MiddlewareServer
 from utils import testgen
 from utils.version import current_version
 
 pytestmark = [
-    pytest.mark.usefixtures('single_middleware_provider'),
+    pytest.mark.usefixtures('setup_provider'),
     pytest.mark.uncollectif(lambda: current_version() < '5.7'),
 ]
 pytest_generate_tests = testgen.generate(testgen.provider_by_type, ["hawkular"], scope="function")
+ITEMS_LIMIT = 5  # when we have big list, limit number of items to test
 
 
-def test_list_deployments(provider):
-    """Tests deployments list between UI, DB and Management system
-    This test requires that no any other provider should exist before.
+def test_list_deployments():
+    """Tests deployments list between UI and DB
 
     Steps:
         * Get deployments list from UI
         * Get deployments list from Database
-        * Get deployments list from Management system(Hawkular)
-        * Compare size of all the list [UI, Database, Management system]
+        * Compare size of all the list [UI, Database]
     """
     ui_deps = _get_deployments_set(MiddlewareDeployment.deployments())
     db_deps = _get_deployments_set(MiddlewareDeployment.deployments_in_db())
-    mgmt_deps = _get_deployments_set(MiddlewareDeployment.deployments_in_mgmt(provider=provider))
```

Comment:
@hhovsepy `mgmt` check will be covered in `test_list_provider_deployments`.
Here testing between `UI` and `DB`. Later we will add `REST`